jmix-core/core/src/main/java/io/jmix/core/FetchPlanBuilder.java

```diff
@@ -36,17 +30,11 @@
  * <p>
  * Use {@link FetchPlans} factory to get the builder.
  */
-@Component("core_FetchPlanBuilder")
-@Scope(BeanDefinition.SCOPE_PROTOTYPE)
 public class FetchPlanBuilder {
 
-    @Autowired
-    protected ApplicationContext applicationContext;
-    @Autowired
+    protected FetchPlans fetchPlansFactory;
```

Comment:
Let's make the variable name just `fetchPlans`. In the framework code we try to give a variable the same name the type has unless there is a specific reason not to do it. And anyway, it is a factory for the FetchPlanBuilder, not for the FetchPlan

Reply:
I agree with these considerations. I used a non-standard name because the `FetchPlanBuilder` class already contains `fetchPlans` variable that is used in many places. Should I rename the variable (to `fetchPlansMap`, for example) and use `fetchPlans` name for the dependency?

Reply:
Okay, I see.. Well, in that case I would go with renaming the collection. Let's make it `propertiesToFetchPlans`.

Reply:
Done.